docs/resources/grant_ownership.md

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
one more thing: we should document the behavior for pipes, so that our users know how to use them. I think it would be best to add it already.

Reply:
Added a section in the `/tmp/grant_ownership.md.tmpl`